core/src/main/java/com/gentics/mesh/core/endpoint/microschema/MicroschemaEndpoint.java

```diff
@@ -193,4 +195,24 @@ private void addCreateHandler() {
 			crudHandler.handleCreate(wrap(rc));
 		}, isOrderedBlockingHandlers());
 	}
+
+	private void addMiscHandlers() {
+		InternalEndpointRoute readOne = createRoute();
+		readOne.path("/:microschemaUuid/projects");
+		readOne.addUriParameter("microschemaUuid", "Uuid of the microschema.", MICROSCHEMA_UUID);
+		readOne.method(GET);
+		readOne.description("Load the projects, attached to the microschema with the given uuid.");
+		readOne.exampleResponse(OK, projectExamples.getProjectListResponse(), "Loaded projects.");
+		readOne.addQueryParameters(GenericParametersImpl.class);
```

Comment:
Endpoint also supports the PagingParameters (at least the paging part), so we should add them here also.